packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -141,3 +141,21 @@ def test_files_api(dcos_api_session):
                 app_task['slaveId'], marathon_framework_id, app_task['id'], required_sandbox_file)
 
             assert content, 'File {} should not be empty'.format(required_sandbox_file)
+
+
+def test_if_ucr_app_runs_in_new_pid_namespace(dcos_api_session):
+    app, test_uuid = get_test_app_in_ucr()
+
+    ps_output_file = 'ps_output'
+    ps_cmd = 'ps ax -o pid= > {};'.format(ps_output_file)
+
+    app['cmd'] = ps_cmd + app['cmd']
+
+    with dcos_api_session.marathon.deploy_and_cleanup(app):
+        marathon_framework_id = dcos_api_session.marathon.get('/v2/info').json()['frameworkId']
+        app_task = dcos_api_session.marathon.get('/v2/apps/{}/tasks'.format(app['id'])).json()['tasks'][0]
+
+        content = dcos_api_session.mesos_sandbox_file(
+            app_task['slaveId'], marathon_framework_id, app_task['id'], ps_output_file)
+
+        assert len(content.split()) < 5, 'UCR app has more than than 5 processes running in its pid namespace'
```

Comment:
Technically, you're checking `< 5`, so an app with 5 processes would fail, which isn't what the error message says.
Either do `<= 5` or update the message to `5 or more processes`.
Also, s/than than/than/